data/xml/J79.xml

```diff
@@ -5,335 +5,903 @@
+  <title>The FINITE STRING, Volume 14, Number 4</title>
+  <month>September</month>
+  <year>1977</year>
+  <abstract>
+    <fixed-case>COLING</fixed-case>-78: Call for Papers (A. Zampolli);
+    Metaphor And Thought: A Conference;
+    Conference on Algorithms for Image and Scene Analysis (K. S. Fu; A. S. Rosefeld);
+    Recent Conferences;
+    <fixed-case>NFAIS</fixed-case> Notes;
+    <fixed-case>N</fixed-case>orthwestern Merges <fixed-case>EE</fixed-case> and <fixed-case>CS</fixed-case> (Stephen S. Yau);
+    <fixed-case>NHPRC</fixed-case> Grant Information;
+    <fixed-case>ICCP</fixed-case> Announces Certificate in Computer Programming;
+    <fixed-case>ACM</fixed-case> Employment Register 1978;
+    <fixed-case>MARSHA</fixed-case>, the Daughter Of <fixed-case>ELIZA</fixed-case> (John K. Cipolaro; Nicholas V. Findler);
+    <fixed-case>AFIPS</fixed-case> Presidential Address, 1976 <fixed-case>NCC</fixed-case>, Dr. Anthony Ralston;
+    <fixed-case>AFIPS</fixed-case> Press Publicatons 1977-1978;
+    <fixed-case>AFIPS</fixed-case> <fixed-case>W</fixed-case>ashington Report (Pender M. McCarter);
+    Linguistic Institute 1978
+  </abstract>
 </paper>
 
 <paper id="1067">
-  <title><i>A Goal Oriented Model of Human Dialogue</i> (<fixed-case>J</fixed-case>ames A Moore; <fixed-case>J</fixed-case>ames A Lenin; William C <fixed-case>M</fixed-case>ann)</title>
+  <title>A Goal Oriented Model of Human Dialogue</title>
+  <author><first>James A.</first><last>Moore</last></author>
+  <author><first>James A.</first><last>Levin</last></author>
+  <author><first>William C.</first><last>Mann</last></author>
 </paper>
 
 <paper id="1068">
-  <title><i>Aspects Of <fixed-case>E</fixed-case>nglish Sentence Stress</i> (Susan F Schmerling), <i>Annual Review Of Information Science And Technology</i> (Ralph Weischedel), <i>Factors Influencing the Placement of english Adverbs in relation to Auxillaries</i> (Ivar Tonisson), <i>Fondazione Dalle Molle Bibliography 1977</i>, <i>New Journal Linguisticea Investigationes</i> (John Benjamins B V), <i>New Journal Studies in langauge</i> (John Benjamins B V), <i>Arts And Humanties Citation Index</i> (Susan Deutch), <i>Current Bibliography</i></title>
+  <title>The FINITE STRING, Volume 14, Number 5</title>
+  <month>September</month>
+  <year>1977</year>
+  <abstract>

```

Comment:
Same here (Abhängigkeitsgrammatik). But it seems weird to have this in J79.xml. I guess this touches on the issue you mentioned about how you dated FINITE STRING issues.

Reply:
This is just because I haven't deleted J79.xml yet.